webui/Caddyfile-dev

```diff
@@ -9,7 +9,15 @@ proxy / localhost:2015 {
 
 proxy /auth keycloak:8080 {
   transparent
-  header_upstream -Authorization
+}
+
+proxy /api api:80 {
+  without /api
+  transparent
+}
+
+proxy /api/files havenapi:3000 {
+  transparent
```

Comment:
now instead of making calls to api server running on a different port, the webui container will proxy traffic from /api and /api/files to the appropriate backend microservices. this way we have a single point where traffic flows into the application.